### PR TITLE
BXMSDOC-2697: Showcase draft

### PR DESCRIPTION
**Pull request by gsheldon**

### CODE REVIEW

File: docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-deploy-itorders-in-showcase-proc.adoc [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
++
+Click *Start* to start the case instance.
+. Select the case from the *Case List* to open the *Overview* page.
++
+From the *Overview* page, you can monitor the case progress, add comments, start new dynamic tasks and processes, and complete and close cases. 
++
+
+[NOTE]
+====
+Cases can be started and closed using the Showcase application, but they cannot be reopened using this application. You can only reopen a case using a Java API call.
```
Comment (mswiderski) [inline docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-deploy-itorders-in-showcase-proc.adoc]: either java or Remote (REST/JMS) API

File: docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-install-and-login-to-showcase-proc.adoc [hunk 2]
```diff
@@ -1,2 +1,33 @@
...
+[WARNING]
+====
+Ensure the names of the {PRODUCT} deployments you are copying do not conflict with your existing deployments in the {EAP} instance
+====
++
+. Add the following system property to your deployment's `__EAP_HOME__/jboss-eap-7.1/standalone/configuration/standalone-full.xml` file:
++
+`<property name="org.jbpm.casemgmt.showcase.url" value="/rhpam-case-mgmt-showcase"/>`
+. Deploy {PRODUCT} using the following command:
++
```
Comment (mswiderski) [inline docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-install-and-login-to-showcase-proc.adoc]: looks like the command is missing?

File: docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-install-and-login-to-showcase-proc.adoc [hunk 3]
```diff
@@ -1,2 +1,33 @@
-[id='']
-= TODO
\ No newline at end of file
+[id='case-management-install-and-login-to-showcase-proc-{context}']
+= Install and login to the Showcase application
+
+The Showcase application is included in the {PRODUCT} {VERSION} distribution in the `add-ons` repository. The purpose of this application is to demonstrate the functionality of case management in {PRODUCT} and allow you to deploy cases created in {CENTRAL}.
```
Comment (cristianonicolai) [inline docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-install-and-login-to-showcase-proc.adoc]: perhaps use _allow you to interact_. Showcase itself does not deploy anything.

File: docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-showcase-application-con.adoc [hunk 4]
```diff
@@ -1,2 +1,8 @@
-[id='']
-= TODO
\ No newline at end of file
+[id='case-management-showcase-application-con-{context}']
+= Case management Showcase application 
+
+The Showcase application is included in the {PRODUCT} distribution to demonstrate the capabilities of case management in an application environment. Showcase is intended to be used as a proof of concept that aims to show the interaction between business process management (BPM) and case management. You can use the application to deploy, monitor, and interact with cases that are being worked on in {CENTRAL}.
```
Comment (cristianonicolai) [inline docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-showcase-application-con.adoc:4]: Perhaps use _create, monitor, and interact with_ ?

File: docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-showcase-application-con.adoc [hunk 5]
```diff
@@ -1,2 +1,8 @@
...
+
+Showcase must be installed in addition to the {CENTRAL} application and {KIE_SERVER}. The Showcase application is required to start new case instances, however the case work is still performed in {CENTRAL}. 
+
+Once a case instance is deployed and being worked on, you can monitor the case in the Showcase by opening the case *Overview* page.
```
Comment (cristianonicolai) [inline docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-showcase-application-con.adoc:8]: is created? Or you mean a case project is deployed?

File: docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-deploy-itorders-in-showcase-proc.adoc (hunk 1 above)
Comment (gsheldon) [reply docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-deploy-itorders-in-showcase-proc.adoc]: ack.

File: docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-install-and-login-to-showcase-proc.adoc (hunk 2 above)
Comment (gsheldon) [reply docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-install-and-login-to-showcase-proc.adoc]: changed with Emily's updates.

Comment (cristianonicolai) [reply docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-install-and-login-to-showcase-proc.adoc]: You also need to at least set the _org.kie.server.location_ system property in order for the showcase to connect with the Kie Server.
The login module `<login-module code="org.kie.security.jaas.KieLoginModule"  flag="required" module="deployment.kie-wb.war"/>`
Module name depends on two things:
 - If you are deploying the showcase in the case server as Business Central you would use: `module="deployment.business-central.war"`
- If standalone deployment of showcase, then the showcase war name: `module="deployment.rhpam-case-mgmt-showcase.war"`

It is also required as well as a user with the following roles:
- kie-server role at the kie server end, so it allows to interact with the remote end points
- user role at the showcase end, which allows the user to login in the app.

File: docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-install-and-login-to-showcase-proc.adoc (hunk 3 above)
Comment (gsheldon) [reply docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-install-and-login-to-showcase-proc.adoc]: changed in epic

File: docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-showcase-application-con.adoc (hunk 4 above)
Comment (gsheldon) [reply docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-showcase-application-con.adoc:4]: done in epic branch.

File: docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-showcase-application-con.adoc (hunk 5 above)
Comment (gsheldon) [reply docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-showcase-application-con.adoc:8]: done in epic branch.

**Pull request merged**